packages/itwinui-react/src/core/utils/components/PopoverCopy.tsx

```diff
@@ -143,6 +220,12 @@ export const PopoverCopy = React.forwardRef((props, ref) => {
     </Box>
   );
 
+  React.useEffect(() => {
+    if (reference) {
+      popover.refs.setReference(reference.current);
```

Comment:
learned recently that this useEffect can be avoided because there is a simpler way to set `reference` too
https://floating-ui.com/docs/react#external-elements